alacritty/src/event.rs

```diff
@@ -190,7 +190,7 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
     /// Request a redraw.
     #[inline]
     fn mark_dirty(&mut self) {
-        self.dirty = true;
+        *self.dirty = true;
     }
```

Comment:
As a matter of good abstraction, I'd expect this to be the only place you need to update the dereference, then all the other spots should just call this function.

Reply:
I've considered this, but a dereference really isn't some complicated magical thing that justifies abstraction.

The main reason for this method is to expose it to `input.rs`, being able to see **all** the code that is executed rather than having to follow a method that contains the code doesn't seem worth it when it's just a single line. So I think it's a little nicer to just inline manually here to make that more transparent.

Reply:
Just hope we never need to add any logic to `fn mark_dirty`. :stuck_out_tongue_winking_eye:

Reply:
Yeah, but we can open that can when it gets to it.